modules/tinymce/src/themes/silver/main/ts/ui/window/SilverDialogEvents.ts

```diff
@@ -33,7 +33,9 @@ const initCommonEvents = <A, S extends EventSpec<A>>(fireApiEvent: FireApiFunc<A
   fireApiEvent<FormCloseEvent>(formCloseEvent, (_api: A, spec: S, _event, self) => {
     // TINY-9148: Safari scrolls down to the sink if the dialog is selected before removing,
     // so we should blur the currently active element beforehand.
-    Focus.active(SugarShadowDom.getRootNode(self.element)).fold(Fun.noop, Focus.blur);
+    if (Focus.hasFocus(self.element)) {
+      Focus.active(SugarShadowDom.getRootNode(self.element)).fold(Fun.noop, Focus.blur);
```

Comment:
Maybe change this to a `each` while you are at it.

Reply:
Can we add a test to this?

Reply:
As noted in the PR message the tests would be very fickle and I couldn't get them to work reliably.